rpmlint/checks/PkgConfigCheck.py

```diff
@@ -0,0 +1,35 @@
+import re
+import stat
+
+from rpmlint.checks.AbstractCheck import AbstractFilesCheck
+
+
+class PkgConfigCheck(AbstractFilesCheck):
+    def __init__(self, config, output):
+        super().__init__(config, output, r'.*/pkgconfig/.*\.pc$')
+        # currently causes too many failures (2008-03-05)
+        self.suspicious_dir = re.compile(r'[=:](?:/usr/src/\w+/BUILD|/var/tmp|/tmp|/home)')
```

Comment:
could be as a class variable and you could kill the init.
also any idea uf the comment makes sense after 10 years?

Reply:
I recommend to remove the comment and we'll see how much it will generate errors.